### DISCUSSION TITLE
Hospedagem da aplicação

### DISCUSSION THREAD

**Discussion by megazordy** [techexpertspro/portal-ascepa#8] [General]
Locais para hospedar a aplicação:

- Levar em conta que o sistema é pra uma ONG, focar melhor custo beneficio.
 
Locais levantados até o momento:
- Vercel no plano free
- Firebase no plano free

Qual sua opinião ou sugestão sobre?

**Comment by andrewarosario** [comment]
Esta semana subirei um deploy da aplicação pela Vercel para entendermos o que ela pode oferecer de forma gratuita.
A grande vantagem será a possibilidade de ter o Deploy Preview, que é um deploy temporário para que todos possam testar uma branch antes de realizar o merge

**Reply by andrewarosario** [reply]
PR criado: #49

**Comment by paoloalmeida** [comment]
Faz total sentido, por enquanto seguirmos com a publicação free na Vercel, em paralelo estou vendo possibilidade de patrocínio para esta hospedagem. Ótima discussão @willflame!
